### PR TITLE
VP-73: Velocloud docs omit instructions for adding plugin to features…

### PR DESCRIPTION
**Pull request by Bonrob2**
…Boot.d

updated to add instructions for adding plugin to featuresBoot.d

### CODE REVIEW

File: docs/modules/installation/pages/installation.adoc [hunk 1]
```diff
@@ -36,4 +36,8 @@ Replace `<VERSION>` with the version you built and want to deploy.
 admin@opennms()> feature:repo-add mvn:org.opennms.plugins.velocloud/karaf-features/<VERSION>/xml
 admin@opennms()> feature:install opennms-plugins-velocloud
 ```
+To ensure that the feature is activated on future service restarts, add the feature to a file in `featuresBoot.d`:
 
+
+[source, console]
+echo "opennms-plugins-velocloud wait-for-kar=opennms-plugins-velocloud" | sudo tee ${OPENNMS_HOME}/etc/featuresBoot.d/plugin-velocloud.boot
```
Comment (emarsh1995) [inline docs/modules/installation/pages/installation.adoc:43]:
```suggestion
[source, shell]
echo "opennms-plugins-velocloud wait-for-kar=opennms-plugins-velocloud" | sudo tee ${OPENNMS_HOME}/etc/featuresBoot.d/plugin-velocloud.boot
```

File: docs/modules/installation/pages/installation.adoc [hunk 2]
```diff
@@ -36,4 +36,8 @@ Replace `<VERSION>` with the version you built and want to deploy.
 admin@opennms()> feature:repo-add mvn:org.opennms.plugins.velocloud/karaf-features/<VERSION>/xml
 admin@opennms()> feature:install opennms-plugins-velocloud
 ```
+To ensure that the feature is activated on future service restarts, add the feature to a file in `featuresBoot.d`:
```
Comment (emarsh1995) [inline docs/modules/installation/pages/installation.adoc:39]:
```suggestion

To ensure that the feature is activated on future service restarts, add the feature to a file in `featuresBoot.d`:
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
Some small formatting updates. Thanks!

**Review by christianpape — APPROVED** [review]
LGTM.

**Review by emarsh1995 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**